docs/src/api/class-apiresponseassertions.md

```diff
@@ -0,0 +1,94 @@
+# class: APIResponseAssertions
+* langs: js
+
+The [APIResponseAssertions] class provides assertion methods that can be used to make assertions about the [APIResponse] in the tests. A new instance of [APIResponseAssertions] is created by calling [`method: PlaywrightAssertions.expectAPIResponse`]:
+
+```js
+import { test, expect } from '@playwright/test';
+
+test('navigates to login', async ({ page }) => {
+  // ...
+  const response = await page.request.get('https://playwright.dev');
+  expect(response).toBeOK();
+});
+```
+
+```java
+...
+import static com.microsoft.playwright.assertions.PlaywrightAssertions.assertThat;
+
+public class TestPage {
+  ...
+  @Test
+  void navigatesToLoginPage() {
+    ...
+    APIResponse response = page.request().get('https://playwright.dev');
+    assertThat(response).isOK();
+  }
+}
+```
+
+```python async
+import re
+from playwright.async_api import Page, expect
+
+async def test_navigates_to_login_page(page: Page) -> None:
+    # ..
+    response = await page.request.get('https://playwright.dev')
+    expect(response).to_be_ok()
+```
+
+```python sync
+import re
+from playwright.sync_api import Page, expect
+
+def test_navigates_to_login_page(page: Page) -> None:
+    # ..
+    response = page.request.get('https://playwright.dev')
+    expect(response).to_be_ok()
+```
+
+
+## method: APIResponseAssertions.not
+* langs: java, js
+- returns: <[APIResponseAssertions]>
+
+Makes the assertion check for the opposite condition. For example, this code tests that the response status is not successfull:
+
+```js
+expect(response).not.toBeOK();
+```
+
+```java
+assertThat(page).not().isOK();
```

Comment:
Should this supposed to assert `response`?

Reply:
yep